kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/service/impl/CreditMemoServiceImpl.java

```diff
@@ -265,4 +266,22 @@ protected void populateDocumentFromPO(VendorCreditMemoDocument cmDocument, HashM
         cmDocument.setUseTaxIndicator(purchaseOrderDocument.isUseTaxIndicator());
     }
 
+    protected void populateItemLinesFromPreq(VendorCreditMemoDocument cmDocument, HashMap<String, ExpiredOrClosedAccountEntry> expiredOrClosedAccountList) {
```

Comment:
I can't see that you changed anything in this method from the delivered code. Is this method override just to bring in the edu.arizona versions of CreditMemoItem and PaymentRequestItem BOs?